### ISSUE TITLE
Fix spell check to detect Spanish spelling errors

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#5007]
Follow up to:

- #5005

Spelling check failed, but was not reported as a failure :-/ ... see https://github.com/open-telemetry/opentelemetry.io/actions/runs/10317103275/job/28560650923

```
Run streetsidesoftware/cspell-action@v6
cspell-action
Pull Request
Error: Configuration
        name: Configuration Loader Error
        msg: Failed to resolve configuration file: "@cspell/dict-es-es/cspell-ext.json" referenced from "./.cspell.yml"
```

Here are the current errors:

```console
$ npm run check:spelling                          

> check:spelling
> npx cspell --no-progress -c .cspell.yml content data 'layouts/**/*.md'

content/es/_index.md:4:3 - Unknown word (Telemetría)
content/es/_index.md:4:59 - Unknown word (observabilidad)
content/es/_index.md:48:54 - Unknown word (telemetría)
content/es/_index.md:62:28 - Unknown word (telemetría)
content/es/_index.md:63:1 - Unknown word (reenvíalos)
content/es/_index.md:80:60 - Unknown word (observabilidad)
CSpell: Files checked: 1312, Issues found: 6 in 1 file.
```

### Tasks

- [x] #5008
- [x] Fix misspelled words - #5009

/cc @theletterf

**Comment by chalin** [comment]
@theletterf - I'm willing to fix the workflow issues. I'll let you address the spelling errors.

**Comment by theletterf** [comment]
Oh whoops. Going to fix that!

**Comment by chalin** [comment]
Oh, this has been failing even since we added pt-br spell-check support! Upstream issue:

- https://github.com/streetsidesoftware/cspell-action/issues/1897

**Comment by chalin** [comment]
Closed by #5008 and #5009
